package.json

```diff
@@ -19,28 +19,28 @@
     "preview-site": "npm run build --prefix sites/svelte-5-preview",
     "check": "cd packages/svelte && pnpm build && cd ../../ && pnpm -r check",
     "format": "prettier --write .",
-    "lint": "prettier --check . && eslint ./",
+    "lint": "cd packages/svelte && pnpm exec rollup -c && cd ../../ && prettier --check . && eslint ./",
```

Comment:
Any idea why we didn't have to do that before?

I think rather than doing `pnpm exec rollup -c && cd ../../` it would probably be nicer to do `pnpm --filter ./packages/svelte build`

Reply:
Hmm. Elsewhere when we have things like tests that only work once the code is built, we don't include automatically building as part of running the script because it gets annoying to have that extra step slowing things down. It is more confusing for newcomers, but it's the trade-off we've generally made to keep things fast for the people working on the code the most